### ISSUE TITLE
getVolume resets iOS AudioSessionCategory to ambient

### ISSUE THREAD

**Issue by meowofficial** [yosemiteyss/flutter_volume_controller#76]
As far as I understand in swift code in some methods default AudioSessionCategory is used instead of configured one.

```
await FlutterVolumeController.setIOSAudioSessionCategory(
  category: AudioSessionCategory.playback,
);

print(await FlutterVolumeController.getIOSAudioSessionCategory()); // AudioSessionCategory.playback

await FlutterVolumeController.getVolume();

print(await FlutterVolumeController.getIOSAudioSessionCategory()); // AudioSessionCategory.ambient
```

**Comment by yosemiteyss** [comment]
Fixed in 1.3.2.

**Comment by meowofficial** [comment]
@yosemiteyss `getVolume` still resets current `AudioSessionCategory` to `AudioSessionCategory.ambient` if it set by another library.

**Comment by kalismeras61** [comment]
This is still an issue!

**Comment by yosemiteyss** [comment]
This issue should be fixed in 1.3.3.
